stock_picking_return_restricted_qty/tests/test_stock_picking_return_restricted_qty.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import UserError
+from odoo.tests import Form, common
+
+
+class StockPickingReturnRestrictedQtyTest(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Remove this variable in v16 and put instead:
+        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+        DISABLED_MAIL_CONTEXT = {
```

Comment:
You can inherit from `BaseCommon` class instead. See https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/tests/common.py#L21

Reply:
fixed, thanks